docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,208 @@
+
+  ![MSFT Device Login](./01.png)
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Create Service Principal](./02.png)
+
+  > *Note : It is highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).*
+
+* Create an Azure resource group where you want the PMK cluster to show up.
+
+  ```shell
+  az group create -l <Azure Region> -n <resource group name>
+  ```
+
+  See below example of creating a resource group.
+
+  ![Create Resource Group](./03.png)
+
+## Deployment
+
+This deployment consists of 3 parts.
+
+  1. Onboarding the node to Platform9 Management Plane.
+  2. Creating a PMK Cluster.
+  3. Connect the cluster to Microsoft Azure Arc.
+
+**Onboarding the node to Platform9 Management Plane**
+
+* Login to your Management Plane.
+
+  ![PMK Management Plane Login Page](./04.png)
+
+  > **If you do not have a registered Management Plane with Platform9, you can create one easily using [PMK Free Tier deployment](https://platform9.com/managed-kubernetes/)**
+
+* Onboard the Ubuntu host to your Management Plane.
+
+  For a BareOS cluster, you will need to have the nodes registered with the PMK Management Plane on which the cluster is to be deployed. For this, first add the node.
+
+  ![Onboard a node](./05.png)
+
+  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to setup the nodes and get connected with Management Plane. Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
+
+  ![PF9 CLI](./06.png)
+
+  > **Note: Preparing the node and connecting it to Management Plane might take approximately 4-5 minutes to complete.**
+
+
+**Creating a PMK Cluster**
+
+* The onboarded node should now be seen on our Management Plane. Now let's create a [PMK cluster](https://platform9.com/learn/learn/get-started-bare-metal) using that onboarded node.
+
+  The cluster creation is done from the PMK Management Plane UI. The steps for cluster creation will follow as below,
+
+  i. Click to add cluster to the Management Plane.
+
+  ![Add Cluster](./07.png)
+
+  ii. Create a cluster from the nodes onboarded to the Management Plane.
+
+  ![Create One Click Cluster](./08.png)
+
+  iii. Enter the cluster name and k8s version. Submit the page and it will trigger the cluster creation on the node.
+
+  ![Cluster name entry](./09.png)
+
+  iv. The cluster should get created in a few minutes and the status should be reported as **Healthy**.
+
+  ![Cluster Created](./10.png)
```

Comment:
```suggestion
- The onboarded node should now be seen on our Management Plane. Now let's create a [PMK cluster](https://platform9.com/learn/learn/get-started-bare-metal) using that onboarded node.

  The cluster creation is done from the PMK Management Plane UI. The steps for cluster creation will follow as below,

  - Click to add the cluster to the Management Plane.

  ![Add Cluster](./07.png)

  - Create a cluster from the nodes onboarded to the Management Plane.

  ![Create One Click Cluster](./08.png)

  - Enter the cluster name and k8s version. Submit the page and it will trigger the cluster creation on the node.

  ![Cluster name entry](./09.png)

  - The cluster should get created in a few minutes and the status should be reported as **Healthy**.

  ![Cluster Created](./10.png)
```